android/sound.cpp

```diff
@@ -130,24 +123,15 @@ void CSound::printStreamDetails ( oboe::ManagedStream& stream )
     QString sSharingMode            = ( stream->getSharingMode() == oboe::SharingMode::Exclusive ? "Exclusive" : "Shared" );
     QString sDeviceID               = QString::number ( stream->getDeviceId() );
     QString sSampleRate             = QString::number ( stream->getSampleRate() );
-    QString sAudioFormat            = ( stream->getFormat()==oboe::AudioFormat::I16 ? "I16" : "Float" );
+    QString sAudioFormat            = ( stream->getFormat() == oboe::AudioFormat::I16 ? "I16" : "Float" );
     QString sFramesPerCallback      = QString::number ( stream->getFramesPerCallback() );
-    qInfo() << "Stream details: [sDirection: " << sDirection <<
-               ", FramesPerBurst: "            << sFramesPerBurst <<
-               ", BufferSizeInFrames: "        << sBufferSizeInFrames <<
-               ", BytesPerFrame: "             << sBytesPerFrame <<
-               ", BytesPerSample: "            << sBytesPerSample <<
-               ", BufferCapacityInFrames: "    << sBufferCapacityInFrames <<
-               ", PerformanceMode: "           << sPerformanceMode <<
-               ", SharingMode: "               << sSharingMode <<
-               ", DeviceID: "                  << sDeviceID <<
-               ", SampleRate: "                << sSampleRate <<
-               ", AudioFormat: "               << sAudioFormat <<
-               ", FramesPerCallback: "         << sFramesPerCallback << "]";
+    qInfo() << "Stream details: [sDirection: " << sDirection << ", FramesPerBurst: " << sFramesPerBurst << ", BufferSizeInFrames: " << sBufferSizeInFrames
```

Comment:
OK, this is a good test for the .clang-format.  The width is just a little too high - this exceeds Github's unified diff frame width.